workspace/libs/shared/ui/src/lib/modules/dashboard/components/navbar-menu/nav-menu.ts

```diff
@@ -163,27 +148,24 @@ export class DashboardV2Component implements OnChanges {
 
         this.isLogoutButtonVisible = false;
         this.isLogoutMenuItemVisible = true;
+        this.isTopMenuVisible = false;
```

Comment:
I think switch case can be removed and we can do it in following way.
 this.isTopMenuVisible = (this.viewport == PidpViewport.small ||  this.viewport ==PidpViewport.large);
similar approach can be followed for other variables as well.

Reply:
[https://bcgovmoh.atlassian.net/browse/PIDP-952](url)
The above ticket will address this conversation.